### PR TITLE
fixes #10, Add support to interact with private docker registry

### LINKED ISSUE #10: Feature: Add support for private docker repository
Add support to ECS cluster to fetch docker images from a private repository which require authentication.

Acceptance Criteria:

ECS should be able to fetch docker image from a private repository using authentication
ECS should be able to fetch docker image from a public repository which does not require authentication

### PR DESCRIPTION
**Pull request by pradeepbhadani**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file.
...
 
+## [Unreleased]
+
+### Added
+- Support for docker private registry - see [#10](https://github.com/ExpediaInc/apiary-authorization/issues/10).
+
+## [2.0.1] - 2019-03-12
+
+### Added
+- Pin module to use `terraform-aws-provider v1.60.0`
```
Comment (massdosage) [inline CHANGELOG.md]: Was this already released? The `version.tf` with that version is in this PR?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+- Support for docker private registry - see [#10](https://github.com/ExpediaInc/apiary-authorization/issues/10).
```
Comment (massdosage) [inline CHANGELOG.md]: docker -> Docker

File: CHANGELOG.md (hunk 1 above)
Comment (pradeepbhadani) [reply CHANGELOG.md]: yes this was released here https://github.com/ExpediaInc/apiary-authorization/releases/tag/v2.0.1

I have just reformatted the code. functioanlity remain same.

File: CHANGELOG.md (hunk 2 above)
Comment (pradeepbhadani) [reply CHANGELOG.md]: done

Comment (massdosage) [reply CHANGELOG.md]: I think you still need to push it ;)

File: templates.tf [hunk 3]
```diff
@@ -4,12 +4,17 @@
...
+  docker_auth = "${ var.docker_registry_auth_secret_name == "" ? "" : format("\"repositoryCredentials\" :{\n \"credentialsParameter\":\"%s\"\n},",join("\",\"",concat(data.aws_secretsmanager_secret.docker_registry.*.arn)))}"
+}
+
 data "template_file" "ranger_admin" {
   template = <<EOF
 [
   {
     "name": "ranger-admin",
     "image": "${var.ranger_docker_image}:${var.ranger_docker_version}",
+    "${local.docker_auth}"
```
Comment (rpoluri) [inline templates.tf]: I think we need to add this for ranger_usersync template also

**Pull request merged**